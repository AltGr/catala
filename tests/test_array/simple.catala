@Article@

/*
new scope A:
   param x content set int 

scope A:
   def x := [0; 4+5; 8*8]

new scope B:
   param a scope A 
   param y content bool
   param z content bool

scope B:
   def y := exists m in a.x such that m = 9
   def z := for all m in a.x we have m > 0
*/