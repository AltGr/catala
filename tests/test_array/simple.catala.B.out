[RESULT] Computation successful! Results:
[RESULT] y = true
[RESULT] z = false
